### DISCUSSION TITLE
[Feature Request] Wails Support

### DISCUSSION THREAD

**Discussion by cryptofyre** [microsoft/msstore-cli#7] [Ideas]
I think this would make a great addition to the project, as Electron is beginning to become a little annoying to work with from a performance and optimization standpoint and Wails follows a fairly familiar package format to Electron but uses WV2 as a substitute.

https://wails.io
https://github.com/wailsapp/wails

**Comment by azchohfi** [comment]
This is a great suggestion! Moving this to a discussion!

**Comment by azchohfi** [comment]
I'm not familiar with Wails. It looks like an amazing option to build Windows apps!

I've searched at their GitHub repo, and it doesn't seem like they support `msix` packaging, only NSIS:
https://github.com/wailsapp/wails/issues/1135

Is this something possible today, and if so, how? Yes, there are Microsoft tools to do this, but I was wondering if the community came up with a more standard solution already, official or unofficial.
